https://dev.azure.com/home-assistant/Core/_build/results?buildId=43697&view=logs&jobId=8b66295a-05e1-52cf-5640-7a4f4e1975e2&j=8b66295a-05e1-52cf-5640-7a4f4e1975e2&t=6c4465fe-285a-5650-787c-2e3dbe1da192

```homeassistant/components/avri/config_flow.py:8:0: W0611: Unused DOMAIN imported from const (unused-import)
```

--- a/homeassistant/components/avri/config_flow.py
+++ b/homeassistant/components/avri/config_flow.py
@@ -5,6 +5,8 @@ import voluptuous as vol
 from homeassistant import config_entries
 from homeassistant.const import CONF_ID
 
+from .const import DOMAIN  # pylint:disable=unused-import
+
 from .const import (
     CONF_COUNTRY_CODE,
     CONF_HOUSE_NUMBER,

--- a/homeassistant/components/avri/config_flow.py
+++ b/homeassistant/components/avri/config_flow.py
@@ -5,8 +5,6 @@ import voluptuous as vol
 from homeassistant import config_entries
 from homeassistant.const import CONF_ID
 
-from .const import DOMAIN  # pylint:disable=unused-import
-
 from .const import (
     CONF_COUNTRY_CODE,
     CONF_HOUSE_NUMBER,
@@ -14,6 +12,7 @@ from .const import (
     CONF_ZIP_CODE,
     DEFAULT_COUNTRY_CODE,
 )
+from .const import DOMAIN  # pylint:disable=unused-import
 
 DATA_SCHEMA = vol.Schema(
     {

--- a/homeassistant/components/avri/config_flow.py
+++ b/homeassistant/components/avri/config_flow.py
@@ -13,7 +13,6 @@ from .const import (
     CONF_HOUSE_NUMBER_EXTENSION,
     CONF_ZIP_CODE,
     DEFAULT_COUNTRY_CODE,
-    DOMAIN,
 )
 
 DATA_SCHEMA = vol.Schema(

